.github/CODEOWNERS

```
@@ -535,9 +535,12 @@ lerna.json @grafana/frontend-ops
 /pkg/infra/httpclient/httpclientprovider/sigv4_middleware.go @grafana/grafana-partnerships-team
 /pkg/infra/httpclient/httpclientprovider/sigv4_middleware_test.go @grafana/grafana-partnerships-team
 
+# Kind definitions
+/kinds/dashboard @grafana/dashboards-squad
+/kinds/ @grafana/grafana-as-code // as-code is fallback owner
```

CODEOWNERS validator doesn't like the format like that :) 

```suggestion
/kinds/ @grafana/grafana-as-code # as-code is fallback owner
```

---

hoo boy i'm moving sloppy-fast through these PRs today - thanks!